spec.bs

```diff
@@ -941,9 +947,12 @@ On the other hand, methods for getting data from the [=shared storage database=]
     1. Let |resultPromise| be a new [=promise=].
     1. Let |context| be {{WindowSharedStorage}}'s {{Window}}'s [=Window/browsing context=].
     1. If |context| is null, return a [=promise rejected=] with a {{TypeError}}.
+    1. Let |document| be |context|'s [=active document=].
+    1. If |document| is not [=fully active=], return a [=promise rejected=] with a {{TypeError}}.
     1. Let |environment| be |context|'s [=active window=]'s [=relevant settings object=].
-    1. If |environment|'s [=associated document=] is not [=fully active=], return a [=promise rejected=] with a {{TypeError}}.
     1. If the result of running [=determine whether shared storage is allowed=] on |environment| is false, return a [=promise rejected=] with a {{TypeError}}.
+    1. Let |origin| be |document|'s [=url/origin=].
+    1. If the result of executing [=is feature enabled|Is feature enabled in document for origin?=] on "[=PermissionsPolicy/shared-storage-select-url=]", |document], and |origin| is "`Disabled`", return false.
```

Comment:
It's simpler to write `If |document| is not [=allowed to use=] the XXX feature, ...`, which already contains the check for "fully active", and uses document's origin by default.

Same for the other place.

Reply:
Done.